### PR TITLE
Storybook formly dialog demo

### PR DESCRIPTION
**Pull request by yerramshilpa**
## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/documentation/src/lib/storybook/formly/formly-dialog/formly-dialog-download/formly-dialog-download.component.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  SDSAutocompletelConfiguration,
+  SdsDialogService,
+  SDSSelectedItemModel,
+  SelectionMode,
+  TabPanelComponent,
+} from '@gsa-sam/components';
+import { SdsFormlyDialogComponent, SdsFormlyTypes } from '@gsa-sam/sam-formly';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+import { AutocompleteSampleDataService } from './autocomplete-sample.service';
+// import { AutocompleteSampleDataService } from '../../../autocomplete/services/autocomplete-sample.service';
```
Comment (cwolf10) [inline libs/documentation/src/lib/storybook/formly/formly-dialog/formly-dialog-download/formly-dialog-download.component.ts:13]: Remove commented code

File: libs/documentation/src/lib/storybook/formly/formly-dialog/formly-dialog.stories.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  createCodePreviewTabData,
+  generateConfig,
+  generateStackblitzLink,
+} from 'libs/documentation/src/sandbox/sandbox-utils';
+
+import { FormlyDialogIntroductionModule } from './formly-dialog-introduction/formly-dialog-introduction.module';
+
+import { FormlyDialogBasicModule } from './formly-dialog-basic/formly-dialog-basic.module';
+import { FormlyDialogDownloadModule } from './formly-dialog-download/formly-dialog-download.module';
+const disable = {
```
Comment (cwolf10) [inline libs/documentation/src/lib/storybook/formly/formly-dialog/formly-dialog.stories.ts:17]: Remove un-used var

**Review by cwolf10 — COMMENTED** [review]
Couple of quick changes

File: libs/documentation/src/lib/storybook/formly/formly-dialog/formly-dialog.stories.ts (hunk 2 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/storybook/formly/formly-dialog/formly-dialog.stories.ts]: Removed

**Review by cwolf10 — DISMISSED** [review]
LGTM

**Comment by cwolf10** [comment]
Once the conflict is resolved of course

**Review by cwolf10 — APPROVED** [review]
Now LGTM

**Pull request merged**